ui/src/app/views/rename-modal/component.html

```diff
@@ -14,7 +14,7 @@
         <button type="button" class="btn btn-outline" (click)="renaming=false">Cancel</button>
         <button type="button" class="btn btn-primary confirm-rename-btn" id="rename"
                 (click)="this.emitRename(resource);" [clrLoading]="loading" [disabled]="loading">
-            Rename {{resourceType}}
+            Rename Notebook
```

Comment:
If we are making this notebook specific, we should probably rename the directory and component.